recipes/evdev/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "evdev" %}
+{% set version = "1.7.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/evdev-{{ version }}.tar.gz
+  sha256: 0c72c370bda29d857e188d931019c32651a9c1ea977c08c8d939b1ced1637fde
+
+build:
+  skip: true  # [not linux]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  script_env:
+    - "C_INCLUDE_PATH=${BUILD_PREFIX}/x86_64-conda-linux-gnu/sysroot/usr/include"       # [linux64]
+    - "C_INCLUDE_PATH=${BUILD_PREFIX}/aarch64-conda-linux-gnu/sysroot/usr/include"      # [aarch64]
+    - "C_INCLUDE_PATH=${BUILD_PREFIX}/powerpc64le-conda-linux-gnu/sysroot/usr/include"  # [ppc64le]
+    - "C_INCLUDE_PATH=${BUILD_PREFIX}/s390x-conda-linux-gnu/sysroot/usr/include"        # [s390x]

```

Comment:
Aren't these covered by the compiler / sysroot activation scripts? 🤔 Does it work if you remove this?

Reply:
It did not work. This envvar is parsed in a custom way within setup.py.

Reply:
https://github.com/gvalkov/python-evdev/blob/0d496bf8a5bce2d5c60147609cb79df1386dbf23/setup.py#L15

Reply:
@jaimergp

Considering that going down to build_ext is not recommended in pip, this should be the best way.

Reply:
Precisely what I wanted. Doing right away.

Reply:
Perhaps the failing issue is that `activate-binutils_linux-64.sh` is after `script_env`.